components/automate-cluster-ctl/habitat/plan.sh

```diff
@@ -22,8 +22,8 @@ pkg_deps=(
   core/make
   core/curl
   core/rsync
-  core/terraform/0.14.8/20210826165930
-  core/hab/1.6.521/20220603154827
+  core/terraform1
+  core/hab
```

Comment:
@schaudha : why you removed the hab pin version ?

Reply:
@punitmundra It was unknowingly. I just reverted to the pinned version.